src/move-array-item.ts

```diff
@@ -3,8 +3,8 @@ const moveArrayItem = <T>(
   fromIndex: number,
   toIndex: number
 ): T[] => {
-  const newArray = array.slice();
-  newArray.splice(toIndex, 0, ...newArray.splice(fromIndex, 1));
+  const newArray = [...array];
```

Comment:
`arr.slice()` is more performant than `[...arr]`
```suggestion
  const newArray = array.slice();
```